src/main/java/frc/robot/RobotMap.java

```diff
@@ -0,0 +1,12 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+
+public class RobotMap {
+    public static final class Pneumatics {
+        public static final int PCM = 41;
+        public static final PneumaticsModuleType PCM_TYPE = PneumaticsModuleType.CTREPCM;
+        public static final int INTAKE_DEPLOY_PCM_CHANNEL = 0;
```

Comment:
Remember to change this one you have this setup on the actual bot

Reply:
Yes